<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="north">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>430</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>North</string>
    </property>
   </widget>
   <widget class="QPushButton" name="east">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>460</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>East</string>
    </property>
   </widget>
   <widget class="QPushButton" name="south">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>490</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>South</string>
    </property>
   </widget>
   <widget class="QPushButton" name="west">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>West</string>
    </property>
   </widget>
   <widget class="QLabel" name="mapLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>Map</string>
    </property>
   </widget>
   <widget class="QLabel" name="moneyLabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>120</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Money&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="inventoryLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>151</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Inventory&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="shopLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>161</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>Shop</string>
    </property>
   </widget>
   <widget class="QLabel" name="mainLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>841</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="itemSelector">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>340</y>
      <width>261</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="itemLabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>380</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buyButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>170</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Buy item</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tradeButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>340</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Trade / Forge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sellButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>230</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Sell item</string>
    </property>
   </widget>
   <widget class="QLabel" name="actionLabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>280</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="collectButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>170</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Collect resources</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mineButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>230</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Mine minerals</string>
    </property>
   </widget>
   <widget class="QLabel" name="captionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>211</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string>Caption</string>
    </property>
   </widget>
   <widget class="QLabel" name="wordleLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>470</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wordle Game</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wordleEdit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>530</y>
      <width>321</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="aLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>500</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="bLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>500</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="cLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>500</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="dLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>500</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="hLabel">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>500</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="gLabel">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>500</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="fLabel">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>500</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="eLabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>500</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="teleportButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>380</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Teleport to spawn</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>867</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
